### DISCUSSION TITLE
Templating within Template text fields

### DISCUSSION THREAD

**Discussion by jtavan** [reflexsoar/reflex-api#171] [Ideas] [answered]
I'd like a Case Template to be able to pre-define a case title field, and I'd like to be able to use a templating language (Jinja2?) to build my Case Template text fields, with variables within the events available for substitution and iteration. This would let me build beautiful and easy to read case title/description markdown, with enrichment data presented in whatever format suits it.

**Comment by n3tsurge** [comment, marked as answer]
This too is on our internal roadmap, we plan to use mustache {{insert.your.field.here}} and it will hunt for the value(s) from the source event(s)

My expectation is you want to be able to prefix the case with something like `[PHISH] - {{event.title}} to {{event.raw_log.mail_recipient}}` so it would show up like `[PHISH] - Suspicious e-mail subject to brian@reflexsoar.com` or something like that.

**Reply by jtavan** [reply]
Mustache probably meets the need, including iteration to generate tables and the like - sounds good.
